src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsServiceOptions.cs

```diff
@@ -55,5 +56,11 @@ public ApplicationInsightsServiceOptions()
         /// Gets or sets a value indicating whether a logger would be registered automatically in debug mode.
         /// </summary>
         public bool EnableDebugLogger { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether a JavaScript to track current authenticated user would be printed along with main
```

Comment:
nitpick grammar: "Gets or sets a value indicating whether a JavaScript snippet to track the current authenticated user should be printed along with the main ApplicationInsights tracking script."